README.md

```diff
@@ -39,7 +39,7 @@ Also read about:
 
 ## Usage
 
-### Warning: 🚨 Marked does not [sanitize](https://marked.js.org/#/USING_ADVANCED.md#options) the output HTML by default 🚨
+### Warning: 🚨 Marked does not [sanitize](https://marked.js.org/#/USING_ADVANCED.md#options) the output HTML. Please use a sanitize library, like [DOMPurify](https://github.com/cure53/DOMPurify) (recommended), [sanitize-html](https://github.com/apostrophecms/sanitize-html) or [insane](https://github.com/bevacqua/insane) on the output HTML! 🚨
```

Comment:
Could we have this list in one place and just link to it. I feel like it will be hard to keep these lists in sync if we ever have to update them.

Reply:
Any suggestion where would you like to put that list _exactly_? It's your project afterall and I feel like this is the point where this change has nothing to do with the security fix.

Also there are already two README.mds (`/README.md` and `/docs/README.md`) with similar content, so you have to keep those in sync too...

Reply:
@koczkatamas: Thank you so much for this!

@styfle: Where would the best place be to store this list? Thinking it would be a nice list for users moving forward as we are planning to get out of the sanitizer business and working looking for things like this anyway. (I have a note to catch up on what I've missed with all the weirdness of life soon.)